pkg/authorization/authorizer/authorizer.go

```diff
@@ -509,21 +521,42 @@ func GetBootstrapPolicyBinding(masterNamespace string) *authorizationapi.PolicyB
 		LastModified: util.Now(),
 		PolicyRef:    kapi.ObjectReference{Namespace: masterNamespace},
 		RoleBindings: map[string]authorizationapi.RoleBinding{
-			"Components": {
+			"system:components": {
```

Comment:
maybe add "-binding" to the binding names to keep them distinct and make sure we're not misusing a binding name as a policy name somewhere accidentally?